File: Sources/hrt.dal/src/entities/Candidate.js


class CandidateEntity {

    get CandidateID() { return this._candidateId; }
    set CandidateID(val) { this._candidateId = val; }

    get FirstName() { return this._fname; }
    set FirstName(val) { this._fname = val; }

    get MiddleName() { return this._mname; }
    set MiddleName(val) { this._mname = val; }

    get LastName() { return this._lname; }
    set LastName(val) { this._lname = val; }

    get Email() { return this._email; }
    set Email(val) { this._email = val; }

    get Phone() { return this._phone; }
    set Phone(val) { this._phone = val; }

    get CVLink() { return this._cvlink; }
    set CVLink(val) { this._cvlink = val; }

    get CreatedDate() { return this._createdDate; }
    set CreatedDate(val) { this._createdDate = val; }

    get CreatedByID() { return this._createdById; }
    set CreatedByID(val) { this._createdById = val; }

    get ModifiedDate() { return this._modifiedDate; }
    set ModifiedDate(val) { this._modifiedDate = val; }

    get ModifiedByID() { return this._modifiedById; }
    set ModifiedByID(val) { this._modifiedById = val; }
}

module.exports = CandidateEntity;

File: Sources/HiringTracker.API/HRT.DTO/Candidate.cs


using System.Text.Json.Serialization;

namespace HRT.DTO
{
    public class Candidate : HateosDto
    {
				[JsonPropertyName("ID")]
		public System.Int64? ID { get; set; }

				[JsonPropertyName("FirstName")]
		public System.String FirstName { get; set; }

				[JsonPropertyName("MiddleName")]
		public System.String MiddleName { get; set; }

				[JsonPropertyName("LastName")]
		public System.String LastName { get; set; }

				[JsonPropertyName("Email")]
		public System.String Email { get; set; }

				[JsonPropertyName("Phone")]
		public System.String Phone { get; set; }

				[JsonPropertyName("CVLink")]
		public System.String CVLink { get; set; }

				[JsonPropertyName("CreatedByID")]
		public System.Int64 CreatedByID { get; set; }

				[JsonPropertyName("CreatedDate")]
		public System.DateTime CreatedDate { get; set; }

				[JsonPropertyName("ModifiedByID")]
		public System.Int64? ModifiedByID { get; set; }

				[JsonPropertyName("ModifiedDate")]
		public System.DateTime? ModifiedDate { get; set; }

				
    }
}
